README.md

```diff
@@ -166,6 +166,23 @@ For more information, see:
 [Metrics API design]: https://github.com/kubernetes/design-proposals-archive/blob/main/instrumentation/resource-metrics-api.md
 [Metrics Server design]: https://github.com/kubernetes/design-proposals-archive/blob/main/instrumentation/metrics-server.md
 
+This diagram shows how metrics-server handles a `kubectl top pods` request:
+```mermaid
+sequenceDiagram
+    participant User
+    participant APIServer
+    participant MS as Metrics-server
+    participant KL as Kubelet
+
+
+    User->>APIServer: GET /apis/metrics.k8s.io/v1beta1/pods
+    APIServer->>MS: List(ctx, opts)
```

Comment:
Why switch from `GET /url` to `LIST(opts)`? Apiserver just passes down the same `GET /apis/metrics.k8s.io/v1beta1/pods` request.